en/getting_started/deploying-on-heroku.md

```diff
@@ -0,0 +1,140 @@
+# Deploying on Heroku
+
+## Manual deployment
+
+This tutorial assumes that you have already managed to clone Consul on your machine and gotten it to work.
+
+1. First, create a Heroku account if it isn't already done.
+2. Install the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) and sign in using
+
+   ```
+   heroku login
+   ```
+
+3. Go to your consul repository and instantiate the process
```

Comment:
Replace `consul` with `CONSUL`

Reply:
Debatable, the main repository (and its forks too, usually) is named 'consul/consul' 😛

Reply:
:thinking: fair enough, I think we haven't quite decided which should be the norm for this :stuck_out_tongue:

That being the case, you could disregard my comments altogether :bowing_man: